src/components/CategoryPage.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const CategoryPage = () => (
+  <div>
+    <button type="button">CHECK STATUS</button>
+  </div>
```

Comment:
- [ ] Kindly be noticed that the button `CHECK STATUS` should dispatch the `underConstruction` action creator.

Reply:
It means when the `CHECK STATUS` button is clicked the action `bookStore/categories/CHECK_STATUS` should be dispatched.

Reply:
I know it is working but you should dispatch this action as you dispatch the create book action when the Add book button clicked.

Reply:
It's not fair as long as it is not in the project requirement.

Reply:
![Screenshot from 2022-06-15 22-17-47](https://user-images.githubusercontent.com/33752735/173919483-db2771c5-9128-4bd1-9b4d-62f31fcacfcb.png)